tests/test_main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.main import check_password
+
+
+def test_too_short_password():
+    assert check_password("qwerty") is False
+
+
+def test_too_long_password():
+    assert check_password("ExtraLongPassword") is False
+
+
+def test_deprecated_symbols():
+    assert check_password("*password") is False
+
+
+def test_is_necessary_symbols_included():
```

Comment:
Please, add tests without capital letters, without special symbols to test all requirements are checked